deeplabcut/generate_training_dataset/labeling_toolbox.py

```diff
@@ -867,14 +873,51 @@ def saveEachImage(self):
             self.dataFrame.loc[self.relativeimagenames[self.iter]][
                 self.scorer, bp[0][-2], "y"
             ] = bp[-1][1]
+	
+    def ResetEachImage(self):
+	    """
+	    Reset data for each image
+	    """
+	    for idx, bp in enumerate(self.updatedCoords):
+	        self.dataFrame.loc[self.relativeimagenames[self.iter]][
+	            self.scorer, bp[0][-2], "x"
+	        ] = None
+	        self.dataFrame.loc[self.relativeimagenames[self.iter]][
+	            self.scorer, bp[0][-2], "y"
+	        ] = None
+    
+
+    def deleteImage(self, event):
+    	image_path = os.path.join( self.currentDirectory, self.dataFrame.index[self.iter])
```

Comment:
I would use self.dataFrame.loc[self.relativeimagenames[self.iter]] like you did above rather than from the DataFrame index directly. This protects us against cases (we recently had) of images named in a way other than imgXXX, which resulted in the ascending order being different in the pandas index and the list of image names.